index.html

```diff
@@ -180,7 +192,39 @@
 
 </tr>
 </table>
+    <div class="form">
+    <form action="/" method="GET" oninput="x.value=parseInt(a.value)">
+        <fieldset>
+            <legend>My first form</legend>
+            <label>Name:<br></br>
+                <input type="text" name="name" placeholder="Name" required></label><br></br>
+            <label>E-mail:<br></br>
+            <input type="email" name="email" required></label><br></br>
+            <label>Password:<br></br>
+            <input type="password" name="password" required></label><br></br>
+            <label>Your programming level<br></br>
+            <input type="range" name="a" value="0" min="0" max="10"></label>
+            <output name="x" for="a"></output><br></br>
+            <p>What does HTML stand for?</p>
+            <label><input type="checkbox" name="check" value="1"> Hyperlink and Text Markup Language</label><br></br>
+            <label><input type="checkbox" name="check" value="2"> Home Tool Markup Language</label><br></br>
+            <label><input type="checkbox" name="check" value="3"> Hyper Text and Markup Language</label><br></br>
+            <p>What does HTTP stand for</p>
+            <label><input type="radio" name="radio" value="1"> Hyper Text Translate Protocol</label><br></br>
+            <label><input type="radio" name="radio" value="2"> Hyper Text Transmission Protocol</label><br></br>
+            <label><input type="radio" name="radio" value="3"> Hyper Text Transfer Protocol</label><br></br>
+            <label>Comment:<br></br><textarea name="message" rows="2" cols="30">Comment</textarea></label><br></br>
+            <label>What does HTML stand for?<br></br><select id="select" name="choice">
+                <option value="answer">Your answer</option>
+            </select>
+            </label><br></br>
```

Comment:
@Hashlemon, I mean close single tag like this <BR/> - not like this <BR></BR> - and generally, as I have already sad, it's not important at all.